rdwatch/urls.py

```diff
@@ -4,16 +4,16 @@
 from django.urls import include, path
 
 urlpatterns = [
-    path('api/', include('rdwatch.urls')),
+    path('api/', include('rdwatch.core.urls')),
     path('admin/', admin.site.urls),
     path('accounts/', include('allauth.urls')),
 ]
 
 # Conditionally add the scoring URLs if the scoring app is installed
-if 'rdwatch_scoring' in apps.app_configs.keys():
-    urlpatterns.append(path('api/scoring/', include('rdwatch_scoring.urls')))
-if 'rdwatch_smartflow' in apps.app_configs.keys():
-    urlpatterns.append(path('api/smartflow/', include('rdwatch_smartflow.urls')))
+if 'rdwatch.scoring' in apps.app_configs.keys():
+    urlpatterns.append(path('api/scoring/', include('rdwatch.scoring.urls')))
+if 'rdwatch.smartflow' in apps.app_configs.keys():
+    urlpatterns.append(path('api/smartflow/', include('rdwatch.smartflow.urls')))
```

Comment:
This isn't working properly.  A dict of the keys in `app.app_configs.keys()` returns the following:
```
dict_keys(['admin', 'auth', 'contenttypes', 'sessions', 'messages', 'staticfiles', 'gis', 'postgres', 'django_extensions', 'django_celery_results', 'allauth', 'account', 'socialaccount', 'gitlab', 'django_rename_app', 'core', 'scoring', 'smartflow', 'debug_toolbar'])
```
Each of the AppConfigs in there has a 'name' property that would match the 'rdwatch.scoring' or the 'rdwatch.smartflow'.  So either we need a way to set the 'key' or instead we search for the name in the app_configs instead of using the key.  There may be a function of `apps` that may make this easier as well.